RubeesEat.IntegrationTests/SeleniumTests/OpenIDSeleniumTests.cs

```diff
@@ -0,0 +1,15 @@
+using RubeesEat.IntegrationTests.PageObjects;
+
+namespace RubeesEat.IntegrationTests.SeleniumTests;
+
+[TestFixture]
+[Explicit("does not run on teamcity")]
```

Comment:
Why does it not run on teamcity? Is it because we can't reach that external service listed in line 13? Who is responsible for maintaining this service?

I think there's  OpenID stubs for local testing, should ask the LF Team. @a-ctor we should look into this.
cc @Frazerus

Reply:
I set the env variables in teamcity, and the test ran successfully. Until we get the stubs we can do it this way, with auth0